src/main/java/be/syntra/devshop/DevshopFront/services/SearchService.java

```diff
@@ -24,9 +24,15 @@
 
     void setAppliedFiltersToSearchModel();
 
-    void setPriceFilters(BigDecimal minPrice,BigDecimal maxPrice);
+    void setPriceFilters(BigDecimal minPrice, BigDecimal maxPrice);
 
     void setSortingByName();
 
     void setSortingByPrice();
+
+    void setSelectedCategory(String category);
+
+    void setRemainingCategories(String category);
+
+    void deleteSelectedCategory(String category);
```

Comment:
removeFormSelectedCategories() i don't like 'delete' her because we're not deleting a category, we removing one from the selected categories

Reply:
fixed